requirements/tests.in

```diff
@@ -1,7 +1,9 @@
 pip-tools
 pip-sync-faster
-coverage
+pytest-coverage
 pytest
 factory-boy
 hypothesis
+filelock
+pytest-xdist[psutil]
```

Comment:
As in LMS, `filelock` is the thing that we use to prevent multiple processes from all initializing the DB and `pytest-xdist` is the thing that runs tests in parallel.